### ISSUE TITLE
Using Select element in Popover

### ISSUE THREAD

**Issue by nicholaskillin** [juliankrispel/react-text-selection-popover#37]
Hey, so I'm not sure if this is a bug or if I've just got something set up wrong.

I'm passing in a component that is a fairly basic `<select>` node with an `onChange` handler setup. 

```
<Popover isOpen={isTextSelected}>
    <CommentCard
      characteristics={characteristics}
      handleCharacteristicSelection={handleCharacteristicSelection}
    />
</Popover>
```

```
function CommentCard({ characteristics, handleCharacteristicSelection }) {
  return (
    <Select
      emptyValue="Characteristic"
      tootip={{ title: 'Select a characteristic' }}
      onChange={({ value }) => handleCharacteristicSelection(value)}
      defaultValue={'Characteristic'}
    >
      {characteristics.map((characteristic) => (
        <Select.Option
          value={characteristic.id}
          children={characteristic.name}
          key={characteristic.id}
        />
      ))}
    </Select>
  )
}
```

When I load my page, the first time I use it everything works great. However, after I've used it once it seems like the `onChange` listener isn't firing anymore.

Here is a link to a gif showing the behavior
https://pic.pco.bz/L1up4vXm

Any idea's what might be going on?

**Comment by esmaeilpour** [comment]
@nicholaskillin 
Add `style={{pointerEvents: 'auto'}}` to the element  and hopefully it will work :)

**Comment by juliankrispel** [comment]
@nicholaskillin @esmaeilpour I've just released 2.0 which lets you implement yourself how to render the popover. Lmk if that solves your issues, closing for now.
